test/UserService.Business.UnitTests/Gender/CreateGenderCommandTests.cs

```diff
@@ -0,0 +1,147 @@
+﻿using FluentValidation.Results;
+using LT.DigitalOffice.Kernel.Helpers.Interfaces;
+using LT.DigitalOffice.Kernel.Responses;
+using LT.DigitalOffice.Kernel.Validators.Interfaces;
+using LT.DigitalOffice.UnitTestKernel;
+using LT.DigitalOffice.UserService.Business.Commands.User;
+using LT.DigitalOffice.UserService.Business.Commands.User.Interfaces;
+using LT.DigitalOffice.UserService.Data.Interfaces;
+using LT.DigitalOffice.UserService.Mappers.Db.Interfaces;
+using LT.DigitalOffice.UserService.Models.Db;
+using LT.DigitalOffice.UserService.Models.Dto.Requests.Gender;
+using LT.DigitalOffice.UserService.Models.Dto.Requests.Gender.Filters;
+using LT.DigitalOffice.UserService.Validation.Gender.Interfaces;
+using Microsoft.AspNetCore.Http;
+using Moq;
+using Moq.AutoMock;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace LT.DigitalOffice.UserService.Business.UnitTests.Gender
+{
+  internal class CreateGenderCommandTests
+  {
+    private AutoMocker _mocker;
+    private ICreateGenderCommand _command;
+    private CreateGenderRequest _request;
+    private DbGender _dbGender;
+    private OperationResultResponse<Guid?> _badResponse;
+    private OperationResultResponse<Guid?> _goodResponse;
+
+    [OneTimeSetUp]
+    public void OneTimeSetUp()
+    {
+      _dbGender = new()
+      {
+        Id = Guid.NewGuid(),
+        Name = "Name",
+        CreatedBy = Guid.NewGuid(),
+        CreatedAtUtc = DateTime.UtcNow
+      };
+
+      _badResponse = new()
+      {
+        Body = null,
+        Errors = new List<string>() { "Error" }
+      };
+
+      _goodResponse = new()
+      {
+        Body = _dbGender.Id,
+        Errors = new List<string>()
+      };
+
+      _request = new()
+      {
+        Name = _dbGender.Name
+      };
+    }
+
+    [SetUp]
+    public void SetUp()
+    {
+      _mocker = new AutoMocker();
```

Comment:
May be not create this object every time?

Reply:
in this case Verifi do not resets(